torchaudio/functional.py

```diff
@@ -449,3 +449,176 @@ def mu_law_expanding(x_mu, qc):
     x = ((x_mu) / mu) * 2 - 1.
     x = torch.sign(x) * (torch.exp(torch.abs(x) * torch.log1p(mu)) - 1.) / mu
     return x
+
+
+def batched_stft(waveforms, fft_length, hop_length=None, win_length=None, window=None,
```

Comment:
I think we can rename this back to stft, since [torchaudio's istft also supports batching](https://github.com/pytorch/audio/blob/master/torchaudio/functional.py#L151). Eventually this should be lowered down to the core of PyTorch, but we might also find [a more general solution to batching](https://github.com/pytorch/pytorch/issues/22169) beforehand.

Until then, for consistency, let's rename this back to stft and create documentation (flags etc.) in compliance with our istft.